Mathlib/Algebra/GroupWithZero/Units/Basic.lean

```diff
@@ -90,6 +90,8 @@ theorem IsUnit.ringInverse {x : M₀} (h : IsUnit x) : IsUnit (inverse x) :=
   match h with
   | ⟨u, hu⟩ => hu ▸ ⟨u⁻¹, (inverse_unit u).symm⟩
 
+@[deprecated (since := "2025-04-22")] alias IsUnit.ring_inverse := IsUnit.ringInverse
```

Comment:
This one looks superfluous. Am I missing something?

Reply:
Hmm, looks like a big in `scripts/add_deprecations.sh`

Reply:
Ahah! Turns out we had both `IsUnit.ring_inverse` and `Ring.IsUnit.ringInverse` 🤦